src/lib/dai.c

```diff
@@ -145,6 +145,9 @@ const struct device *zephyr_dev[] = {
 #if CONFIG_DAI_INTEL_HDA
 	DT_FOREACH_STATUS_OKAY(intel_hda_dai, GET_DEVICE_LIST)
 #endif
+#if CONFIG_DAI_NXP_SAI
```

Comment:
Is this specific to imx93?
I don't think so, therefore please update commit message.

Reply:
fixed